docs/manual/publish.md

```diff
@@ -42,7 +42,25 @@ For example,
 <iframe src="http://< ip-address >:< port >/#/notebook/2B3QSZTKR/paragraph/...?asIframe" height="" width="" ></iframe>
 ```
 
+Don't forget to also add below lines in your page for loading the Twitter Bootstrap themes and CSS used for Zeppelin notebook style, or your paragraph will be shown ugly.
+
+```
+<head>
+     ....
+    <link rel="stylesheet" href="zeppelin-web/bower_components/bootstrap/dist/css/bootstrap.css" />
+    <link rel="stylesheet" href="zeppelin-web/fonts/font-awesome.min.css">
+    ....
+</head>
+```
+
 Finally, you can show off your beautiful visualization results in your website. 
 <center><img src="../assets/themes/zeppelin/img/docs-img/your-website.png" height="90%" width="90%"></center>
 
 > **Note**: To embed the paragraph in a website, Zeppelin needs to be reachable by that website. 
+
+## Troubleshooting
```

Comment:
I don't think this should be here,
We should add something saying the feature is not working, in the 0.5.5 doc